contracts/ComptrollerG6.sol

```diff
@@ -1280,7 +1280,7 @@ contract ComptrollerG6 is ComptrollerV5Storage, ComptrollerInterface, Comptrolle
      * @return The amount of COMP which was NOT transferred to the user
      */
     function grantCompInternal(address user, uint amount) internal returns (uint) {
-        Comp comp = Comp(getCompAddress());
+        INV comp = INV(getCompAddress());
```

Comment:
Is this functionality needed? Isn't this for COMP rewards? Are we mirroring that with INV?

Reply:
yes we are